src/components/AudioPlayer/AudioPlayer.module.scss

```diff
@@ -52,6 +52,9 @@
     flex-direction: inherit;
```

Comment:
There should be 3 visibility states.

refactor `isMinimized` to `visibility`

Visibility will take one of 3 values: 

- Default (default) 
- Minimized
- Expanded